tests/core/test_accounts.py

```diff
@@ -434,31 +419,44 @@ def test_eth_account_recover_transaction_from_eth_test(acct, transaction):
     assert acct.recoverTransaction(raw_txn) == expected_sender
 
 
-@pytest.mark.parametrize(
-    'private_key, password, kdf, expected_decrypted_key, expected_kdf',
-    [
+def encrypt_mark_params():
+    """
+    Params for testing Account#encrypt. Due to not being able to provie fixtures to
+    pytest.mark.parameterize, we opt for creating the params in a non-fixture method
+    here instead of providing fixtures for the public key, password, and private key.
+    """
+    public_key = '0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318'
```

Comment:
This is a private key, the only difference between `web3js_key` and `web3js_private_key` was that one is a hex string and the other is a `keys.PrivateKey`. So the variable name should be updated.